app/assets/javascripts/channels/room.coffee

```diff
@@ -0,0 +1,13 @@
+App.room = App.cable.subscriptions.create "RoomChannel",
+  connected: ->
+    # Called when the subscription is ready for use on the server
+
+  disconnected: ->
+    # Called when the subscription has been terminated by the server
+
+  received: (data) ->
+      console.log("Response: " + data["message"]);
```

Comment:
Will this affect all users, or is it turned "on" for just some?

Reply:
As I said in the other comment. Room channel will be connected to all the sessions.